### PR TITLE
React devtools implementation

### PR DESCRIPTION
**Pull request by SounD120**
Added React Devtools element inspector to the command pallete #791

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -87,6 +88,10 @@
 			{
 				"command": "reactNative.reloadApp",
 				"title": "React Native: Reload App"
+			},
+			{
+				"command": "reactNative.runInspector",
+				"title": "React Native: Element Inspector"
```
Comment (ruslan-bikkinin) [inline package.json]: I would change this to "React Native: Run Element Inspector" for more transparency to user.

Comment (SounD120) [reply package.json]: Yes, I think it can improve the user experience.

File: src/extension/commandPaletteHandler.ts [hunk 2]
```diff
@@ -210,6 +215,31 @@ export class CommandPaletteHandler {
...
+            if (fs.existsSync(devToolsPath)) {
+            CommandPaletteHandler.elementInspector = spawn(electron, [devToolsPath], {
+                env: {
+                    PATH: process.env.PATH,
+                },
+            });
+            CommandPaletteHandler.elementInspector.once("exit", () => {
+                CommandPaletteHandler.elementInspector = null;
+            });
+    } else {
```
Comment (ruslan-bikkinin) [inline src/extension/commandPaletteHandler.ts]: Please, fix indentations

File: test/extension/elementInspector.test.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            CommandPaletteHandler.runElementInspector();
+            assert.notEqual(CommandPaletteHandler.elementInspector, null);
+
+            if (CommandPaletteHandler.elementInspector) {
+                CommandPaletteHandler.elementInspector.once("exit", () => {
+                    assert.equal(CommandPaletteHandler.elementInspector, null);
+                    done();
+                });
+            } else
+                assert.ifError("element inspector didn't launch properly");
```
Comment (ruslan-bikkinin) [inline test/extension/elementInspector.test.ts]: Please always use bracets `{ }` even for single line statements

Comment (SounD120) [reply test/extension/elementInspector.test.ts]: Thanks for pointing out, i'll fix this.

**Pull request merged**